app/src/main/java/org/simple/clinic/bp/history/BloodPressureHistoryScreenUi.kt

```diff
@@ -0,0 +1,8 @@
+package org.simple.clinic.bp.history
+
+import org.simple.clinic.bp.BloodPressureMeasurement
+
+interface BloodPressureHistoryScreenUi {
+  fun showBloodPressureHistory(bloodPressures: List<BloodPressureMeasurement>)
+  fun showNoBloodPressuresView()
```

Comment:
I think this method was missed when removing the show empty view code.

Reply:
Fixed